docs/tools/concepts.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Wallet Concepts
+hide_title: false
+---
+Vega Wallet is an app that lets you manage wallets and cryptographic key pairs. Key pairs are used to sign and send transactions in a secure way. 
+
+Read on to understand the basic concepts of wallets and keys, and how they work. 
+
+## Cryptographic keys
+Each cryptographic key pair includes a public key and a private key. Those keys are associated with an entity that needs to authenticate their identity, or sign and send transaction data. Data that is encrypted with the public key can only be decrypted with its corresponding private key.
+
+Anyone can send someone encrypted information using the public key associated with the private key. However, only the owner of the private key can decrypt the information. The goal is to prove that a transaction was indeed signed by the originator and was not forged.
```

Comment:
Based on my previous comment, this explanation should be rework for to explain the "signature" process instead.

Reply:
Reworked this - please re-read!